src/System.Linq/tests/ToArrayToListToDictionaryTests.cs

```diff
@@ -0,0 +1,469 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+
+namespace System.Linq.Tests
+{
+    public class ToArrayToListToDictionaryTests
```

Comment:
These tests use the same set of helper classes. So it will require to put helper classes in separate files too. Is it OK?